src/components/charts/brightnessVsDistance/Points.jsx

```diff
@@ -35,6 +35,8 @@ class Points extends React.PureComponent {
             [notActive]: (selectedData || hoveredData) && !selected && !hovered,
             [invisible]: x === null || y === null,
           });
+          const blueColorPercent = Math.floor((color / 2) * 100);
+          const redColorPercent = 100 - blueColorPercent;
```

Comment:
Great trick.  I love the results, and so does Lauren.  I asked her to think about if there are any sciency color cutoffs we should consider, or sciency hues of red and blue.  If we end up wanting to assert even more control over how to translate these values into visual colors I'm sure we can abstract the work I did in the HR Diagram to translate temps into colors (see `charts/scatter-plot/points.jsx`)

Reply:
Thank you!!! I was more than excited when I put this together. Very happy with the results.